README.md

```diff
@@ -1,6 +1,24 @@
 # Computest Openrunner
 [![Build Status](https://travis-ci.org/computestdev/Openrunner.svg?branch=master)](https://travis-ci.org/computestdev/Openrunner) [![Coverage Status](https://coveralls.io/repos/github/computestdev/Openrunner/badge.svg?branch=master)](https://coveralls.io/github/computestdev/Openrunner?branch=master) [![Greenkeeper badge](https://badges.greenkeeper.io/computestdev/Openrunner.svg)](https://greenkeeper.io/) [![Known Vulnerabilities](https://snyk.io/test/github/computestdev/Openrunner/badge.svg)](https://snyk.io/test/github/computestdev/Openrunner)
 
-Benchmark and functional testing for frontend-heavy web applications
 
-More documentation will follow soon
+
+Openrunner can be used for benchmark and functional testing for frontend-heavy web applications. It's a tool that simulates an end user using a website. It simulates user behaviour (keyboard/mouse activity) to browse through an online application. This can be used to test functionality and/or response times. Openrunner is a browser extension but can also be run from the command line i.e. for integration in a build pipeline.
+
+Getting stared:
+
+First of all, browse to https://github.com/computestdev/Openrunner/releases to download the latest release. Currently, releases come in form of a profile for Firefox. Since our plugin is not an official browser plugin yet you'll have to download Firefox Nightly: https://nightly.mozilla.org/. 
+
+Once you have both files installed/extracted you can run Firefox with Openrunner from the command line: "<path to firefox executable> --no-remote --profile <profile directory>"
```

Comment:
Could use backticks here instead of quotes. That way it will render monospaced and special characters such as `<...>` will not be interpeted